### PR TITLE
Complete The Design of The Pages

### PR DESCRIPTION
**Pull request by batool2azzam**
This pull request completes the design and implementation of the registry page. It will also be dedicated to completing the rest of the pages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,10 +13,14 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/material": "^5.15.18",
+    "formik": "^2.4.6",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.51.4",
```
Comment (IbrahimLama) [inline package.json]: why this library is installed? I don't see its usage

File: src/pages/Register/Register.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                label="First Name"
+                margin="normal"
+                variant="filled"
+                id="firstName"
+                name="firstName"
+                value={formik.values.firstName}
+                onChange={formik.handleChange}
+                onBlur={formik.handleBlur}
+                error={
+                  formik.touched.firstName && Boolean(formik.errors.firstName)
```
Comment (IbrahimLama) [inline src/pages/Register/Register.tsx]: can be written 
!!(formik.errors?.firstName)

Comment (batool2azzam) [reply src/pages/Register/Register.tsx]: when I write it "!!(formik.errors?.firstName)" then all fields return error when I toch one field , like this screenShot
Should I use the same phrase or keep it as it was?
image

File: package.json (hunk 1 above)
Comment (batool2azzam) [reply package.json]: I was using UseForm Hook and then converted to Formik. I will delete it

File: src/pages/Register/Register.tsx (hunk 2 above)
Comment (IbrahimLama) [reply src/pages/Register/Register.tsx]: it should be 
{ formik.touched.firstName && !!(formik.errors.firstName)}

!! should convert any value to its corresponding boolean value

File: package.json (hunk 1 above)
Comment (IbrahimLama) [reply package.json]: yes uninstall it, you should also uninstall "@hookform/devtools": "^4.3.1",

**Pull request merged**